src/sentry/data_export/processors/discover.py

```diff
@@ -21,20 +21,20 @@ class DiscoverProcessor:
     Processor for exports of discover data based on a provided query
     """
 
-    def __init__(self, organization_id, discover_query):
-        self.projects = self.get_projects(organization_id, discover_query)
-        self.environments = self.get_environments(organization_id, discover_query)
+    def __init__(self, organization, discover_query):
```

Comment:
Actual changes here, had to change the param from organization_id to the organization object.

And changed `get_environments` to return a list of objects instead of names